src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -129,7 +136,7 @@ export class BrowsingContextProcessor {
   //       "canAccessOpener": false,
   //       "browserContextId": "1B5244080EC3FF28D03BBDA73138C0E2" },
   //     "waitingForDebugger": false } }
-  #handleAttachedToTargetEvent(
+  async #handleAttachedToTargetEvent(
```

Comment:
Async event handler can lead to race condition as we had with realms recently.

Reply:
Done